### PR TITLE
Block props E2E's

### PR DESCRIPTION
**Pull request by mustafauzunn**
**Description**:

E2E tests for block properties

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                        cryptoTransfer(HapiCryptoTransfer.tinyBarsFromTo(GENESIS, FUNDING, 1)),
+
+                        ethereumCall(contract, "logNow")
+                                .type(EthTxData.EthTransactionType.EIP1559)
+                                .signingWith(SECP_256K1_SOURCE_KEY)
+                                .payingWith(RELAYER)
+                                .nonce(1)
+                                .maxFeePerGas(50L)
+                                .gasLimit(1_000_000L)
+                                .via(timeLoggingTxn)
```
Comment (IvanKavaldzhiev) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java:143]: We can name this txn `secondBlock`

Comment (mustafauzunn) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java]: DONE

File: test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+                        uploadInitCode(contract),
+                        contractCreate(contract)
+                ).when(
+                        ethereumCall(contract, "logNow")
+                                .type(EthTxData.EthTransactionType.EIP1559)
+                                .signingWith(SECP_256K1_SOURCE_KEY)
+                                .payingWith(RELAYER)
+                                .nonce(0)
+                                .maxFeePerGas(50L)
```
Comment (shemnon) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java:128]: This should be zero. Unless we are explicitly testing "relayer pays"

Comment (shemnon) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java:128]: This should be failing for insufficient fees.  IIRC gas price is ~71 in test environments.  We should fail this transaction unless there is an auto-allowance from the relayer.

**Review by shemnon — APPROVED** [review]

File: test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java (hunk 2 above)
Comment (mustafauzunn) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java:128]: There is a [default allowance of five hbars](https://github.com/hashgraph/hedera-services/blob/2a89626b0823e3a31c40a6eb4cd7de030dd5a9a7/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/contract/HapiEthereumCall.java#L91) , which should cover the fees if gas price is not sufficient.

**Pull request merged**